### DISCUSSION TITLE
Добавить внешний дашборд на главную страницу?

### DISCUSSION THREAD

**Discussion by khpavel** [totumonline/totum-ru-issues-and-discussions#505] [Как это сделать?] [answered]
Добрый день.

Хочу добавить внешний embedded код на первую страницу totum через тег iframe. 
Totum пропустит?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вроде нет таких ограничений:

<img width="2005" alt="Снимок экрана 2024-01-09 в 14 29 34" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/089e6e96-bc45-455d-86e3-1eaf6bb1d881">

Но может быть запрет со стороны того, что вы собираетесь встраивать.
